docs/content/operations/pull-deps.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: doc_page
+---
+# pull-deps Tool
+
+`pull-deps` is a tool that can pull down dependencies to the local repository and lay dependencies out into the extension directory as needed.
+
+`pull-deps` has several command line options, they are as follows:
+
+`-c` or `--coordinate`     
+
+    Extension coordinate to pull down, followed by a maven coordinate, e.g. io.druid.extensions:mysql-metadata-storage:0.8.0-rc1
+
+`-h` or `--hadoop-coordinate`
+
+    Hadoop dependency to pull down, followed by a maven coordinate, e.g. org.apache.hadoop:hadoop-client:2.4.0
+
+`--no-default-hadoop`
+
+    Don't pull down the default hadoop coordinate, i.e., org.apache.hadoop:hadoop-client:2.3.0. If `-h` option is supplied, then default hadoop coordinate will not be downloaded.
+
+`-clean`
```

Comment:
can we prefix options that have more than one letter with `--` i.e. `--clean` to be consistent?